### ISSUE TITLE
Documentation on Theming

### ISSUE THREAD

**Issue by DapperMongoose** [cogentcore/core#1315]
### Describe the feature

The ability to set the background color via a styler with `s.Background = colors.Scheme.Background` seems to hint that there is a theme system of some sort in place.  I see it referenced briefly in: https://github.com/cogentcore/core/issues/521 which now returns a 404 error.

I would like some documentation on how to set up a theme and what different widgets use by default if possible.  Additionally it would be awesome if these docs included information on how to create multiple themes and switch between them, for instance to allow the user to select a theme or perform the relatively standard light/dark mode toggle.

**Comment by kkoreilly** [comment]
Apologies for the lack of documentation on this; improving the documentation will now be my highest priority. In the meantime, if you right click and then click on Settings, you can change theme settings.

**Comment by ghost** [comment]
It's early days in the framework I know.  Just trying to help call out some high level things that might help out other early adopters. :)

Would the user changing the theme in settings persist between restarts of the application?

**Comment by kkoreilly** [comment]
Yes, the settings are saved to the filesystem when changed and thus persist between sessions.
